src/components/TimePicker.vue

```diff
@@ -124,7 +124,7 @@ export default {
   },
   methods: {
     displayFormattedDateTime(datetimeIsoString) {
-      return DateTime.fromISO(datetimeIsoString).toLocaleString(this.displayFormat)
+      return DateTime.fromISO(datetimeIsoString,{zone:this.$parent.timezone}).toLocaleString(DateTime.DATETIME_FULL)
```

Comment:
pourquoi as-tu retiré `this.displayFormat` ? 
Tu aurais pu ajouter une prop à TimePicker pour recuperer le timezone pour eviter le $parent